### DISCUSSION TITLE
Ability to Add Vertical Threshold lines to Histogram Panel

### DISCUSSION THREAD

**Discussion by gannaramu** [grafana/grafana#54655] [Dashboards, Panels & Visualizations]
Hello Team, Wanted to check if it is possible to create vertical threshold lines on Histogram Panel. Something like this..
<img width="533" alt="image" src="https://user-images.githubusercontent.com/12237488/188187818-7b62f10d-4f66-44e4-925e-322df4ed201e.png">

**Comment by gannaramu** [comment]
@leeoniya would like to know your thoughts on this.

**Reply by leeoniya** [reply]
feels pretty niche to me to justify options growth/complexity. but we can let this discussion simmer for a while to collect feedback. generally this seems close to "time regions" feature that we still have to bring back to the new TimeSeries panel, so maybe it can piggyback on the same core.

https://stackoverflow.com/questions/59848587/is-there-a-way-to-overlay-an-on-off-sensor-over-a-graph-in-grafana

**Comment by engincannot** [comment]
Exactly what I am looking for!

**Comment by katerinaangjeli** [comment]
I'm wondering when this is expected to be rolled out? It would be extremely useful.

**Comment by leeoniya** [comment]
now that we landed https://github.com/grafana/grafana/pull/62273 (see discussion there), this might be worth another look.

we only have thresholds in one direction. so if that direction is x (buckets), it cannot also be y (counts). i'm not sure which is more useful, honestly, but they feel mutually exclusive to me without a clear winner.

wdyt?

**Reply by katerinaangjeli** [reply]
For my use case it would be more useful to get thresholds in the x because it helps visualize data that is outside of relevant standard deviations.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
